packages/select/src/components/select/multiSelect.tsx

```diff
@@ -184,11 +184,12 @@ export class MultiSelect<T> extends React.PureComponent<IMultiSelectProps<T>, IM
             if (this.input != null && this.input !== document.activeElement) {
```

Comment:
@adidahiya for uncontrolled usage, `nextOpenState` is entirely ignored.
Regardless of the value of `nextOpenState`, we close the popover if the input is not active.

Before this change, we'd still forward the (potentially incorrect) value of `nextOpenState` to those with controlled usage (through `onInteraction`).
After this change, we only call `onInteraction` when we'd also call `setState`, and we call with the literal boolean values, indifferent of the `nextOpenState` value.